docs/index.asciidoc

```diff
@@ -14,20 +14,27 @@ ingesting data into Elasticsearch. A common schema helps you correlate
 data from sources like logs and metrics or IT operations
 analytics and security analytics.
 
-ECS is still under development and backward compatibility is not guaranteed. Any
-feedback on the general structure, missing fields, or existing fields is
-appreciated. For contributions please read the
+[[ecs-maturity]]
+=== Maturity
+
+With ECS turning 1.0, the team will will approach improvements by following SemVer.
+This means we will only do breaking changes at major versions, and that
+we will improve ECS in a backwards compatible manner in the meantime.
```

Comment:
improvements will be implemented in a backwards-compatible manner in the meantime.